LibreNMS/OS/AirosAfLtu.php

```diff
@@ -51,22 +57,32 @@ public function discoverWirelessDistance()
      */
     public function discoverWirelessPower()
     {
-        $rx_power0_oid = '.1.3.6.1.4.1.41112.1.10.1.4.1.5.24.232.41.30.48.222'; //UBNT-AFLTU-MIB::afLTUStaRxPower0.1
-        $rx_power1_oid = '.1.3.6.1.4.1.41112.1.10.1.4.1.6.24.232.41.30.48.222'; //UBNT-AFLTU-MIB::afLTUStaRxPower1.1
-        $rx_ideal_power0_oid = '.1.3.6.1.4.1.41112.1.10.1.4.1.7.24.232.41.30.48.222'; //UBNT-AFLTU-MIB::afLTUStaIdealRxPower0.1
-        $rx_ideal_power1_oid = '.1.3.6.1.4.1.41112.1.10.1.4.1.8.24.232.41.30.48.222'; //UBNT-AFLTU-MIB::afLTUStaIdealRxPower1.1
-        $rx_power0_level_oid = '.1.3.6.1.4.1.41112.1.10.1.4.1.9.24.232.41.30.48.222'; //UBNT-AFLTU-MIB::afLTUStaRxPowerLevel0.1
-        $rx_power1_level_oid = '.1.3.6.1.4.1.41112.1.10.1.4.1.10.24.232.41.30.48.222'; //UBNT-AFLTU-MIB::afLTUStaRxPowerLevel1.1
+        $oids = array();
+        $sensors = array();
+
```

Comment:
you could do a walk here and then foreach over the data to add the wireless sensors.

Reply:
If I read the whole table with snmpwalk, I would have to check in foreach if the next value is the one I want to read because the table contains some other values that are not useful. I also don't know why ubiquiti uses a table here at all since the airfiber is a pure point to point bridge. So there can only be one single entry in the table. Or do I have a mistake here?

Reply:
When using some of the functions we have it will group entries together so you get nice output.  snmpwalk_group(), snmpwalk_cache_oid(), snmpwalk_cache_numerical_oid() are some of these functions.

Reply:
The problem is that with none of the snmpwalk functions I got the data I needed. Either I'm doing something wrong or it's because of the way ubnt designed the table. For example with snmpwalk_cache_oid I get an array with the index '...).8.' which I can't work with. If I use snmpwalk_cache_oid_num I get an empty array. snmpwalk_group gives me as index the mac address of the counterpart which is also not useful in this case. Do you have an idea what goes wrong with this?

Reply:
yeah, that index is why the values are wonky.   try setting the option `-Ob`  (set the full snmp_flags: `-OteQUsb`)

Reply:
That's it! Thank you for your help :)